modules/nodes-cluster-worker-latency-profiles-using.adoc

```diff
@@ -138,3 +139,80 @@ $ oc get KubeControllerManager -o yaml | grep -i workerlatency -A 5 -B 5
 
 To change the medium profile to default or change the default to medium, edit the `node.config` object and set the `spec.workerLatencyProfile` parameter to the appropriate value.
 
+[id="nodes-cluster-worker-latency-profiles-using-at-creation_{context}"]
+== Using worker latency profiles at cluster creation
+
+To implement a worker latency profile in the manifest, follow these steps:
+
+* Create the manifest needed to build the cluster.
+* Edit the `node.config` object to add the name of the profile. You can change the profile at any time as latency increases or decreases.
+* Define `workerLatencyProfile` using the `"Default"` profile.
++
+[source,terminal]
+----
+$ cat <cluster-install-dir>/manifests/config-node-default-profile.yaml
+----
++
+.Example output
+[source,terminal]
+----
+apiVersion: config.openshift.io/v1
+kind: Node
+metadata:
+ name: cluster
+spec:
+ workerLatencyProfile: "Default"
+----
+
+[id="nodes-cluster-worker-latency-profiles-examining_{context}"]
+== Example steps for displaying resulting values of 'workerLatencyProfile'
```

Comment:
Same as above, make sure this is okay to do (multiple procedures in one module)

Reply:
I've broken these procedures out into their own modules. Thank you!